package.json

```
@@ -122,7 +123,8 @@
     "validator": "~13.7.0",
     "vite": "~2.9.1",
     "vitepress": "~0.22.3",
-    "vitest": "~0.8.4"
+    "vitest": "~0.8.4",
+    "mime-db": "^1.52.0"
```

Please use 
```suggestion
    "mime-db": "~1.52.0"
```

After that you need to clean and re-install to generate a new lock file

Also you did not use `pnpm add -D mime-db` because it's not on the correct sorted position

---

Nope i didnt call it, i like to open a new pr where i like to call, npx sort-package-json

---

was faster for me, its fixed :)